DataFormats/EgammaCandidates/src/Photon.cc

```diff
@@ -90,6 +90,7 @@ void Photon::setCorrectedEnergy(P4type type, float newEnergy, float delta_e, boo
     case regression1:
       eCorrections_.regression1Energy = newEnergy;
       eCorrections_.regression1EnergyError = delta_e;
+      [[fallthrough]];
```

Comment:
Shouldn't it rather break here? @Sam-Harper

Reply:
yes, this is bug, thanks for pointing this out. Luckily I dont think we use regression1 and its regression2 that is our main correction.  Likewise for comment below